src/ess/sans/normalization.py

```diff
@@ -0,0 +1,63 @@
+import scipp as sc
+import scippneutron as scn
+
+
+def solid_angle(data, pixel_size, pixel_length):
+    """
+    Solid angle function taking pixel_size and pixel_lenght as parameters.
+    The method assumes pixels are rectangular.
+    """
+    L2 = scn.L2(data)
+    return (pixel_size * pixel_length) / (L2 * L2)
+
+
+def covert_and_rebin(data, wavelength_bins, begin, end):
+    """
+    Convertining and rebinning shifted data
```

Comment:
```suggestion
    Convertining and rebinning shifted data
```
This docstring does not say more than the function name? What is the actual function purpose?

Reply:
Before this function was called `setup ` and I thought I can give it more descriptive name. Can revert it to `setup` if it works better.

Reply:
I think if a function is just used within a file then a good name may not be so important, but it seems this is imported elsewhere (`sans.py`) so we probably want something that is truly descriptive?

Reply:
Indeed, it was initially used internally in one file but then it turned out that can be applied elswhere. Maybe `contrib.py `is anyway better place for such helper function?

Reply:
`contrib.py` was menat for things that may eventuallt make it into scipp itself. I think that is quite unlikely for this particular function.